subprojects/packagefiles/bzip2/meson.build

```diff
@@ -0,0 +1,109 @@
+project('bzip2', 'c', version: '1.0.8', meson_version: '>=1.0.0')
+
+
+add_project_arguments('-D_FILE_OFFSET_BITS=64', language: 'c')
```

Comment:
This flag should already be automatically added by meson.

Reply:
TIL, thanks!